guides/v2.2/migration/migration-tool-configure.md

```diff
@@ -11,81 +11,169 @@ menu_order: 4
 
 After you install the data migration tool, the following directory contains mapping and configuration files:
 
-*	{{site.data.var.ce}}:
+* {{site.data.var.ce}}:
 
-	*	`<your Magento 2 install dir>/vendor/magento/data-migration-tool/etc/opensource-to-opensource`: Configuration and scripts for migrating from {{site.data.var.ce}} 1 to {{site.data.var.ce}} 2
+    * `<your Magento 2 install dir>/vendor/magento/data-migration-tool/etc/opensource-to-opensource`: Configuration and scripts for migrating from {{site.data.var.ce}} 1 to {{site.data.var.ce}} 2
 
-*	{{site.data.var.ee}}:
+* {{site.data.var.ee}}:
 
-	*	`<your Magento 2 install dir>/vendor/magento/data-migration-tool/etc/opensource-to-commerce`: Configuration and scripts for migrating from {{site.data.var.ce}} 1 to {{site.data.var.ee}} 2
-	*	`<your Magento 2 install dir>/vendor/magento/data-migration-tool/etc/commerce-to-commerce`: Configuration and scripts for migrating from {{site.data.var.ee}} 1 to {{site.data.var.ee}} 2
+    * `<your Magento 2 install dir>/vendor/magento/data-migration-tool/etc/opensource-to-commerce`: Configuration and scripts for migrating from {{site.data.var.ce}} 1 to {{site.data.var.ee}} 2
+    * `<your Magento 2 install dir>/vendor/magento/data-migration-tool/etc/commerce-to-commerce`: Configuration and scripts for migrating from {{site.data.var.ee}} 1 to {{site.data.var.ee}} 2
 
 The preceding directories contain subdirectories for each supported version.
 
 ## Configuring the migration {#migration-configure}
 
-Before you migrate any data, you must create a `config.xml` configuration file from the provided sample.
-
-To create a configuration file:
+There are two possible ways to configure the Data Migration Tool for migration:
+- change the Data Migration Tool configs in the `<your Magento 2 install dir>/vendor/magento/data-migration-tool/etc/` 
+- configure the Data Migration Tool in the separate module (recommended) 
 
-1.	Log in to your Magento server as, or switch to, the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/apache-user.html).
+In the first case you bringing manual changes to the `<your Magento 2 install dir>/vendor` directory. Ideally, this directory should be modified only by [composer](https://glossary.magento.com/composer). It's not a sort of "good" practice but MUST in case if you use continuous delivery for your environments.
```

Comment:
Sorry, I wasn't clear enough. Let's simplify the explanation using two following cases: 

If you want to put your migration configuration under version control and use it for deployment, you need to create a separate module. 
If you are not going to put your changes under version control and run the Migration tool only locally, you can edit files in the vendor directory directly.

Reply:
**done**

Thank you, @rogyar